### PR TITLE
dont mention interactive; instead mention rx

### PR DESCRIPTION
**Pull request by ahuang11**

### CODE REVIEW

File: examples/tutorial/01_Overview.ipynb [hunk 1]
```diff
@@ -151,11 +151,11 @@
    "source": [
     "Each of the HoloViz tools provides its own API for accessing whatever functionality it implements (and sometimes more than one API, for different purposes!). For this tutorial, we will focus on the following APIs that we believe provide the most power to new users for the smallest investment of time and effort:\n",
     "\n",
-    "* [hvPlot](https://hvplot.pyviz.org) `.plot()` API, which adds additional power to a well-known API supported by many different data libraries. \n",
-    "* [hvPlot](https://hvplot.pyviz.org) `.interactive()` API, which brings live interactivity to the existing Pandas and Xarray APIs.\n",
+    "* [hvPlot](https://hvplot.holoviz.org) `.plot()` API, which adds additional power to a well-known API supported by many different data libraries. \n",
+    "* [Param](https://param.holoviz.org) `.rx()` API, which brings live interactivity to the existing Pandas and Xarray APIs.\n",
```
Comment (maximlt) [inline examples/tutorial/01_Overview.ipynb:155]: Not sure this is how Jim would define `rx()`, he'll anyway be able to update this when he'll review the tutorial.

**Review by maximlt — APPROVED** [review]

File: examples/tutorial/05_Interactive_Pipelines.ipynb [hunk 2]
```diff
@@ -148,7 +148,7 @@
    "id": "04c8c9a9",
    "metadata": {},
    "source": [
-    "What if instead of '5', we want the output above always to reflect the current value of `mag_slider`? We can do that by using hvPlot's `.interactive()` support, passing in a widget almost anywhere we want in a pipeline:"
+    "What if instead of '5', we want the output above always to reflect the current value of `mag_slider`? We can do that by using hvPlot's `.rx()` support, passing in a widget almost anywhere we want in a pipeline:"
```
Comment (maximlt) [inline examples/tutorial/05_Interactive_Pipelines.ipynb]: hvPlot has no `.rx()` support.

**Comment by ahuang11** [comment]
Building a Dashboard still has a mention of `interactive`, but not a priority for scipy tutorial I think

File: examples/tutorial/05_Interactive_Pipelines.ipynb (hunk 2 above)
Comment (ahuang11) [reply examples/tutorial/05_Interactive_Pipelines.ipynb]: Yup; latest changes fixed.

Comment (maximlt) [reply examples/tutorial/05_Interactive_Pipelines.ipynb]: Cool thanks.

**Comment by philippjfr** [comment]
Thanks folks, knew I missed a few.

**Comment by ahuang11** [comment]
Going to merge this for now. Can be updated later.

**Pull request merged**